client/curriculum/html/HTMLStructure/HTMLStructure.tsx

```diff
@@ -369,17 +370,4 @@ function HTMLStructure() {
   );
 }
 
-type ResponsiveFlexProps = PropsWithChildren<HTMLAttributes<HTMLDivElement>>;
```

Comment:
Good job deleting this.

Let's also delete the related styles from `HTMLStructure.module.scss`.

Reply:
Here I left the 
```
.main-structure-wrapper {
  width: 20em;
  margin-left: 5em;
}
```
because I can see that it's used here in the component:
```
 <Highlight
              language="html"
              className={styles['main-structure-wrapper']}
 />
```

Reply:
Great!